src/test/java/calculator/tests/SubtractionTest.java

```diff
@@ -0,0 +1,13 @@
+package calculator.tests;
+
+import org.testng.Assert;
+import org.testng.annotations.Test;
+
+public class SubtractionTest extends GenericTestData {
+
+    @Test
+    public void fiveMinusThree() {
+        long result = calculator.sub(5, 3);
+        Assert.assertTrue(result == 2, "Incorrect result of subtraction");
```

Comment:
'assertEquals' is better in this case

Reply:
Changed. 
Just tried to use different methods.